lib/parsers/livesplit.rb

```diff
@@ -85,6 +85,7 @@ def v1_6(xml, fast)
         raise "Not a proper LiveSplit run file"
       end
 
+      run[:history].unshift(run[:time]) unless fast || run[:time].in?(run[:history])
```

Comment:
Long term, faking history entries might get weird since LiveSplit assigns IDs to each attempt history so you can look up the corresponding segment histories, and we probably want to start scraping and returning those IDs at some point. When that happens faking histories will be a no-no. 

What specific problem does this solve? Is there a different way to solve it?